README.md

```diff
@@ -2,6 +2,18 @@
 
 Substra end-to-end (e2e) tests
 
+# A few words regarding tests design
+
+- Must be fast to run
+- Should be stable: no random failure
+- Require a setup with at least 2 nodes
```

Comment:
A 1 node setup could be enough in some cases (checking that a basic execution works for example)

Reply:
Tests cannot run currently with a 1 node setup, we could make them compatible easily with a bit work.
I was thinking of increasing to a minimal of 3 nodes as it enables to test more use cases.

Reply:
Ok, so no more tests marked as skipped because they require extra nodes then?

Reply:
Exactly. But I think that would be possible only when we will have a default skaffold setup with 3 nodes.

Reply:
👍